s3/template.go

```diff
@@ -106,6 +135,7 @@ You can also raise an issue on the [Export Service GitHub repo](https://github.c
 		meta.ExportOrgID,
 		meta.ExportDate,
 		dataDetails,
+		failedFilesDetails,
```

Comment:
You may want to add a test case with a failed file to make sure this is working as expected